docs/integration/ibm-cloud-kubernetes-logs.md

```diff
@@ -0,0 +1,253 @@
+```
+
+**When you want to edit the config, change the `logagent.conf`, recreate the ConfigMap, restart the Logagent pod to grab the new ConfigMap and you're done!**
+
+Continue reading below to see how to configure more advanced settings.
+
+### Drop Noisy Container Logs 
+
+Edit the `logagent.conf` to add the [drop-events](../logagent/output-filter-dropevents/) outputFilter.
+
+```yaml hl_lines="13 14 15 16 17 18 19"
+# logagent.conf
+options:
+  debug: false
+
+input: 
+  files: 
+    - /var/log/*.log
+    - /var/log/containers/*.log
+
+inputFilter:
+  - module: input-filter-k8s-containerd
+
+outputFilter:
+  dropEvents:
+    module: drop-events
+    filters:
+      message:
+        include: !!js/regexp /critical|auth|error|failed/
+        exclude: !!js/regexp /status/i
+
+output:
+  elasticsearch:
+    module: elasticsearch
+    url: https://logsene-receiver.sematext.com
+    index: YOUR_SEMATEXT_LOGS_TOKEN
+
+```
+
+In the `filters` section you can pick a field to apply regex mathing to.
+If there's a match you can either include or exclude that particular log line.
+The sample above will include all log lines that match `critical|auth|error|failed` but exclude all that match `status`.
+
+### Forward Container Logs to Multiple Apps with Log Routing
+
+To enable log routing you edit the output plugin to use multiple indices.
+Under the token value you need to add a regex for the log file you want to match.
+
+```yaml hl_lines="24 25 26 27 28"
+# logagent.conf
+options:
+  debug: true
+
+input: 
+  files: 
+    - /var/log/*.log
+    - /var/log/containers/*.log
+
+inputFilter:
+  - module: input-filter-k8s-containerd
+
+outputFilter:
+  dropEventsFilter:
+    module: drop-events
+    filters:
+      message:
+        exclude: !!js/regexp /status/i
+
+output:
+  elasticsearch:
+    module: elasticsearch
+    url: https://logsene-receiver.sematext.com
+    indices: 
+      b0e9f481-xxxx-xxxx-xxxx-3ff20227d3d3: # All logs except kube-system
+        - ^(?!.*(kube-system).*).*\.log
+      9365eb2f-xxxx-xxxx-xxxx-5a833072353f: # Only kube-system logs
+        - .*kube-system.*\.log
+
+```
+
+### Enable Kubernetes Audit Logs
+
+Create a Kubernetes Audit Logs App and a Generic Logs App.
+
+![](https://sematext.com/wp-content/uploads/2020/07/apps.sematext.com_ui_logs-create-2.png)
+
+Forwarding Kubernetes audit logs is similar to log routing. The prerequisite is to [follow steps 1-9 in the official IBM docs here](https://cloud.ibm.com/docs/containers?topic=containers-health#webhook_logdna).
+
+This will provision an audit webhook that will forward logs to a log file. With Logagent you can tail this file and forward logs to a Kubernetes Audit Logs App in Sematext.
+
+in your `logagent.conf`, under the `indices` section, where you specify the token value for your Kubernetes Audit Logs App, you need to add a regex for the name of the audit webhook you created in the steps above. The name of this Pod is `ibm-kube-audit`.
```

Comment:
in => In

Reply:
Pod is capitalized here. I saw it all lowercase further above.

Reply:
Capitalized it everywhere. :ok_hand: